src/coreclr/vm/peassembly.inl

```diff
@@ -236,7 +236,8 @@ inline LPCWSTR PEAssembly::GetDebugName()
 #ifdef _DEBUG
     return m_pDebugName;
 #else
-    return GetPath();
+    SString path = GetPath();
+    return path.GetUTF8();
```

Comment:
This is going return dangling pointer.

Reply:
Ugh. Yes it is. I'm going to need to reconsider a few things here anyways.